### DISCUSSION TITLE
能否限制OPENVPN转入流量的IP或IP段

### DISCUSSION THREAD

**Discussion by Wp516781950** [angristan/openvpn-install#892] [Help]
能否限制OPENVPN转入流量的IP或IP段?

**Comment by javasboy** [comment]
This can be handled by the firewall iptables

这个交给防火墙 iptables 处理即可

**Comment by Wp516781950** [comment]
我设置了iptables后发现没效果，这个openvpn-install，安装后怎么重启openvpn服务？

**Comment by Wp516781950** [comment]
seerver.conf : push "route xxx.xxx.xxx.xxx xxx.xxx.xxx.xxx" 并不生效。

**Comment by zhangj3** [comment]
> seerver.conf : push "route xxx.xxx.xxx.xxx xxx.xxx.xxx.xxx" 并不生效。

try to reply in English, 尽量英文回复吧，这个项目的用户/维护者基本都是老外
